### ISSUE TITLE
No way to cancel

### ISSUE THREAD

**Issue by mmavko** [niksy/throttle-debounce#6]
Expected something like this:

```javascript
debouncedF = debounce(1000, F);
debouncedF();
// after 500 ms
debouncedF.cancel();
// not called
```

**Comment by niksy** [comment]
This module is converted version of [jquery-throttle-debounce](https://github.com/cowboy/jquery-throttle-debounce) which doesn’t have cancel method for debounce.

This could be nice addition. I’m willing to accept a PR for this. Maybe something like this is already done for [one of the forks of original module](https://github.com/cowboy/jquery-throttle-debounce/network)?

**Comment by niksy** [comment]
@mmavko @whatknight Released in 2.1.0!
